frontend/public/components/nav.jsx

```diff
@@ -227,6 +229,10 @@ const NavSection = connect(navSectionStateToProps)(
       this.setState({isOpen: expandState});
     }
 
+    getPluginNavItems = memoize(
+      (section) => plugins.registry.getNavItems(section)
```

Comment:
I would prefer to avoid starting the pattern of using stateful globals in a component. Much prefer passing in a prop as it makes components easier to test when they rely only on props.
`withExtensionRegistry` or `withPlugins` HOC

Reply:
Missed reading your `Follow-ups` section.
I'm ok with doing this in stages.

Reply:
> I would prefer to avoid starting the pattern of using stateful globals in a component.

I agree completely :+1:

This is just a temporary step between "no link to plugins" and "React-proper link to plugins".

In a follow-up PR, we will formalize how React components access the plugin registry.